testing/logstash/logstash.conf

```diff
@@ -0,0 +1,12 @@
+input {
+        file {
+        type => "syslog"
+        path => ["/var/log/**/*.log"]
+        }
+}
+
+output {
+        elasticsearch { hosts => ["localhost:9200"] }
```

Comment:
I just used the settings from the [ES reference docs](https://www.elastic.co/guide/en/logstash/current/configuration.html) - have commented out `elasticsearch`.

I also changed the log directory in the `confd` so logstash does not consume it's own logs

By default the `stdout` plugin prints to the `/var/lib/logstash/logs/error.log`)